### PR TITLE
First student version.

### PR DESCRIPTION
**Pull request by vicentgodella**

### CODE REVIEW

**Review by Fsero — APPROVED** [review]

File: infra/aws/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Export an environment variable with your project_code. For example:
+
+`export PROJECT_CODE=cto-bootcamp-student-X`
+
+After that, you can create cloudformation stacks with the following command (inside infra directory):
+
+`sceptre --var "VpcName=<student-name>" launch-env bootstrap`
```
Comment (Fsero) [inline infra/aws/README.md:7]: is not always the bootstrap environment, remember to update the README

File: script/deploy-local-k8s.sh [hunk 2]
```diff
@@ -1,33 +1,41 @@
...
-KUBE_LATEST_VERSION="v1.9.3"
+KUBE_LATEST_VERSION="v1.8.0"
 HELM_VERSION="v2.8.2"
 BIN_DIRECTORY="${SCRIPT_DIR}/bin"
 HELM_PATH=${BIN_DIRECTORY}/helm
 KUBECTL_PATH="${BIN_DIRECTORY}/kubectl"
-CHARTS_PATH="${SCRIPT_DIR}/../helm/charts"
-RELEASE_NAME="example"
+CHARTS_PATH="${SCRIPT_DIR}/../infra/k8s/helm/charts"
+RELEASE_NAME="vicent"
```
Comment (Fsero) [inline script/deploy-local-k8s.sh]: Remember to include an step in the document to change this variable

Comment (Fsero) [reply script/deploy-local-k8s.sh]: It would be better maybe to set an environmental variable

File: infra/aws/README.md (hunk 1 above)
Comment (vicentgodella) [reply infra/aws/README.md:7]: Yep, my idea is that students can follow the readme instructions to get things done.

**Review by Fsero — APPROVED** [review]

**Pull request merged**